estolowka-frontend/src/environments/environment.ts

```diff
@@ -4,5 +4,6 @@
 // The list of which env maps to which file can be found in `.angular-cli.json`.
 
 export const environment = {
-  production: false
+  production: false,
+  baseUrl: "http://" + window.location.hostname + ":8080"
```

Comment:
I would also suggest to change name of url e.g. 'mealUrl' and add "/meal" to address.